sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,151 @@
+        Apply for funding
+      </h2>
+      <content-group>
+        <div class="category-header-medium-small pa-2">
+          Financial Aid Application
+        </div>
+        <div class="pb-6 px-2">
+          Apply to see if you are applicable for provincial or federal loans and
+          grant. This is for students attending full-time or part-time
+          post-secondary studies.
+        </div>
+        <div class="category-header-medium-small pb-3 px-2">
+          Select program year
+        </div>
+        <v-select
+          v-model="programYearId"
+          :items="programYearOptions"
+          variant="solo"
+          density="compact"
+          label="Select year"
+          class="px-2 pb-4"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Year')]"
+          hide-details="auto"
+        ></v-select>
+        <v-btn
+          class="ma-2"
+          variant="elevated"
+          data-cy="primaryFooterButton"
+          color="primary"
+          @click="startApplication"
+          >Start Application</v-btn
+        >
+      </content-group>
+    </student-page-container>
+  </v-form>
 </template>
 <script lang="ts">
-import { ref } from "vue";
+import { onMounted, ref, defineComponent } from "vue";
 import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
-import {
-  useFormioDropdownLoader,
-  useFormioUtils,
-  useSnackBar,
-  useModalDialog,
-} from "@/composables";
+import { useSnackBar, useModalDialog, useRules } from "@/composables";
 import { useRouter } from "vue-router";
-import { FormIOCustomEvent, FormIOCustomEventTypes } from "@/types";
+import { VForm, SelectItemType } from "@/types";
 import { ApplicationService } from "@/services/ApplicationService";
 import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
 import { ProgramYearService } from "@/services/ProgramYearService";
+import ContentGroup from "@/components/generic/ContentGroup.vue";
 
-export default {
-  components: { ModalDialogBase },
+export default defineComponent({
+  components: { ModalDialogBase, ContentGroup },
   setup() {
     const initialData = ref({});
     const router = useRouter();
     const snackBar = useSnackBar();
-    const formioDataLoader = useFormioDropdownLoader();
-    const formioUtils = useFormioUtils();
     const { showDialog, showModal } = useModalDialog<void>();
-    const PROGRAM_YEAR_DROPDOWN_KEY = "programYear";
+    const programYearOptions = ref([] as SelectItemType[]);
+    const programYearId = ref();
+    const startApplicationForm = ref({} as VForm);
+    const { checkNullOrEmptyRule } = useRules();
 
-    const formLoaded = async (form: any) => {
-      await formioDataLoader.loadProgramYear(form, PROGRAM_YEAR_DROPDOWN_KEY);
-    };
-    const dialogClosed = () => {
+    onMounted(async () => {
+      const programYearOptionTmp =
+        await ProgramYearService.shared.getProgramYearOptions();
+      programYearOptions.value = programYearOptionTmp.map((yearOption) => {
```

Comment:
`
programYearOptions.value = programYearOptionTmp.map((yearOption) => ({
        
          title: yearOption.description,
          value: yearOption.id.toString(),
        
      }))
    });
`to simplify. u can you do something like this too